server/api/models/member.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
```

Comment:
i think this models folder will definitely grow in size as we continue developing. Can you look into putting these models in subfolders? e.g server/api/models/user/member.js

Reply:
is everyone in our church considered a member? if so i think it makes more sense to call them user? or something more generic?

if not everyone in our church considered a member, should probably generalise the "user" into something more scalable. 

open to discussing more about this

Reply:
true, it should be more general. updated the schema according to this suggestion and renamed 'member' to 'user'